sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_services_first_seen_v1/query.sql

```diff
@@ -1,136 +1,44 @@
-WITH base AS (
+WITH fxa_events AS (
   SELECT
-    * REPLACE (
-      IF(service IS NULL AND event_type = 'fxa_activity - cert_signed', 'sync', service) AS service
-    )
+    timestamp,
+    user_id,
+    IF(service IS NULL AND event_type = 'fxa_activity - cert_signed', 'sync', service) AS service,
+    os_name,
   FROM
-    firefox_accounts.fxa_content_auth_oauth_events
+    `moz-fx-data-shared-prod.firefox_accounts.fxa_content_auth_oauth_events`
+  WHERE DATE(timestamp) = DATE(@submission_date)
```

Comment:
@kik-kik it looks like the init file creates an empty table and this is running a one day pull. I'm wondering how we do the initial backfill. For the [browser first seen tables](https://github.com/mozilla/bigquery-etl/blob/9bca48821a8a0d40b1700cc14ecd8068d132ed06/sql/moz-fx-data-shared-prod/telemetry_derived/clients_first_seen_v1/query.sql) the process is usually a backfill that scans the entire table then the daily job checks to see if the client_id value already exists in the first seen table then appends the new data only.

Reply:
I'm happy to explain the backfill options to you if you'd like in a call. This is fine.